src/cli.js

```diff
@@ -5,6 +5,12 @@ const { constants, promises: fs } = require("fs");
 const path = require("path");
 const { inject } = require("./api.js");
 
+const logger = {
+  info: (message) => console.log("\x1b[36m%s\x1b[0m", message),
+  success: (message) => console.log("\x1b[32m%s\x1b[0m", message),
+  error: (message) => console.log("\x1b[31mError: %s\x1b[0m", message),
```

Comment:
What would these characters look like on Windows?

Reply:
I could check it on my windows machine.

Reply:
My dear @fraxken just try it: 
![log-windows](https://user-images.githubusercontent.com/22824417/221226219-1b9e9102-629c-4ef9-a9d5-b0ce83971209.png)

Reply:
Also tested on Powershell and classical invite command (look fine).

Reply:
Thanks for the help @fraxken! Does it also work on cmd?

Reply:
Yes it work with cmd too